docs/index.md

```diff
@@ -44,13 +44,9 @@ provider "snowflake" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- `account` (String) The name of the Snowflake account. Can also come from the `SNOWFLAKE_ACCOUNT` environment variable.
-- `username` (String) Username for username+password authentication. Can come from the `SNOWFLAKE_USER` environment variable.
-
 ### Optional
 
+- `account` (String) The name of the Snowflake account. Can also come from the `SNOWFLAKE_ACCOUNT` environment variable.
```

Comment:
This should say that at least one of the following combination is required:
- account + username + password
- account + username + private key etc
- profile